l10n_it_delivery_note/__manifest__.py

```diff
@@ -13,7 +13,7 @@
     "author": "Marco Calcagni, Gianmarco Conte, Link IT Europe Srl, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/l10n-italy",
-    "version": "14.0.1.1.2",
+    "version": "14.0.2.0.0",
```

Comment:
@odooNextev rimuovere

Reply:
In questo caso specifico, per la presenza dello script di migrazione, è ok il bump della versione :+1:

Reply:
Ottimo, c'è sempre da imparare :)